runtime/gc_ctrl.c

```diff
@@ -546,20 +551,23 @@ static void test_and_compact (void)
 CAMLprim value caml_gc_major(value v)
 {
   CAML_INSTR_SETUP (tmr, "");
+  caml_ev_begin(EV_EXPLICIT_GC_MAJOR);
   CAMLassert (v == Val_unit);
   caml_gc_message (0x1, "Major GC cycle requested\n");
   caml_empty_minor_heap ();
   caml_finish_major_cycle ();
   test_and_compact ();
   // call finalisers
   caml_process_pending_actions();
+  caml_ev_end(EV_EXPLICIT_GC_MAJOR);
```

Comment:
Same here

Reply:
diff is outdated (maybe because I force-pushed at some point prior to this changed, but it's now handled, thank you very much!